src/client/routes/manage/CreateSponsor.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import Spinner from '../../components/Loading/Spinner';
 import STRINGS from '../../assets/strings.json';
 
-const EMAIL_REGEX = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+import * as EmailValidator from 'email-validator';
```

Comment:
This should go on line 3. I believe the import sorter will put it in the right place if you eliminate the whitespace on line 17.